developer-docs-site/docs/tutorials/index.md

```diff
@@ -22,6 +22,21 @@ If running macOS, install the following packages in the order specified to take
     ```
 1. **Poetry**: Install Poetry from [https://python-poetry.org/docs/#installation](https://python-poetry.org/docs/#installation).
 
+## Adjust network and funding source
+
+For development purposes, the tutorials here assume you are working in the Aptos devnet network and therefore interacting with the devnet Aptos node and faucet service for creating and funding accounts. As noted in the relevant tutorials, these targets can be configured with the following environment variables:
+  * `APTOS_NODE_URL`
+  * `APTOS_FAUCET_URL`
+
+Here are the options for funding by network:
+  * *Mainnet* - There is no faucet in-wallet; you have to get funds from an exchange.
+  * *Testnet* - Use the faucet button in-wallet or use the testnet faucet at: https://aptoslabs.com/testnet-faucet.
+  * *Devnet* - Use the faucet button in-wallet or use the devnet faucet at: https://faucet.devnet.aptoslabs.com
+
+For more information on the available Aptos networks, see [Aptos Blockchain Deployments](../nodes/aptos-deployments.md).
```

Comment:
I think the reality is that currently the only way to really use these examples end-to-end is with devnet or a local testnet. I think the guidance should be much more explicit.

Reply:
That is -- you can only test these in these two environments, here's how...

Reply:
> That is -- you can only test these in these two environments, here's how...

Great feedback, David. I've attempted to address this both at the top and in this new section. Please take another look when you get a chance. Thanks!